README.md

```diff
@@ -186,7 +186,21 @@ then just `pip install dumb-init`.
 ## Usage
 
 Once installed inside your Docker container, simply prefix your commands with
-`dumb-init`. For example:
+`dumb-init`.
+
+Within a Dockerfile, it's a good practice to use dumb-init as your container's
+entrypoint. An "entrypoint" is a partial command that gets prepended to your
+`CMD` instruction, making it a great fit for dumb-init:
+
+```bash
+ENTRYPOINT ["/usr/bin/dumb-init", "--"]
```

Comment:
shouldn't this be \`\`\`Dockerfile (also ps, you have no idea how long it took me to try and format this damn comment, and I still didn't get it code-formatted)

and perhaps: ```# Runs `/usr/bin/dumb-init -- /my/script --with --args` ```